### DISCUSSION TITLE
Idea - Check spam message user account creation date

### DISCUSSION THREAD

**Discussion by ageekhere** [ThioJoe/YT-Spammer-Purge#629] [Ideas]
I have an idea (not sure if has been added yet), what if we check the spam message user account creation date as part of limiting false positives. If the account was created X years ago it is most likely not going to be a spam account. 

Maybe like an option like search for accounts younger than 1 month old.

I wonder what the average spam account age is.

**Comment by ThioJoe** [comment]
Yea that's something I've considered, but surprisingly, many spam accounts are actually pretty old. I've seen some that are literally a year old. Goes to show how long these spam accounts are able to operate, or at least that the scammers plan far ahead and create new accounts all the time, and might only use older ones.

**Reply by TechStudent10** [reply]
There is the obvious factor that scammers can hack accounts with insecure passwords. It's been seen with Jawed, the oldest account on YouTube.
